packages/ui/src/coreModules/form/components/fields/Date/SingleDate/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import FieldWrapper from '../../../FieldWrapper'
+import SingleDateComponent from './Component'
+import {
+  futureSingleDate,
+  pastSingleDate,
+  textParsable,
+} from '../validationFunctions'
+
+const propTypes = {
+  future: PropTypes.bool,
+  past: PropTypes.bool,
+  validate: PropTypes.array,
+  validateText: PropTypes.bool,
+}
+
+const defaultProps = {
+  future: false,
+  past: false,
+  validate: [],
+  validateText: true,
+}
+
+function SingleDateField(props) {
```

Comment:
mm I think the component file should also be called SingleDateField so it is more clear that this one actually contains the FieldWrapper